topics/migration.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Migration from Redis to Valkey
+linkTitle: Migration
+description: How to migrate from Redis to Valkey
+---
+
+This is a migration guide from Redis Community Edition to Valkey. 
+You will learn how to migrate a standalone Redis server instance and a Redis Cluster. 
+
+This guide provides migration steps for Redis server and Valkey deployed in Docker; however, they should also apply for on-premises deployments. 
+Refer to [install Valkey](installation.md) for installation options.
+
+## Why to migrate to Valkey?
+
+* Valkey is the vendor-neutral and open-source software
+* Enhanced performance with multi-threading and dual-channel replication
+* Improved memory efficiency by using one dictionary per slot in cluster mode and embedding keys in dictionaries. 
+
+### Migration compatibility matrix
+
+You can migrate Redis server to Valkey. 
+The following table provides migration options depending on the Redis version you run:
+
+| Redis | Valkey |
+|-------|--------|
+| 6.x   | 7.2.x  |
+| 7.2.x | 7.2.x  |
+| 7.2.x | 8.0.x  |
+| 7.4   | n/a    |
+
+## Migrate a standalone instance
+
+To migrate a standalone Redis server to Valkey, you have the following options:
+
+* [Physical migration](#physical-migration) by copying the most recent on-disk snapshot from the Redis server to Valkey and starting Valkey server with it
+* [Setting up replication](#repication) between Redis and Valkey 
+* [Migrating specific keys](#migrate-specific-keys) 
+
+The example migration steps are provided for Redis 7.2.5 and Valkey version 7.2.6. 
+
+### Physical migration
+
+This is the easiest and fastest migration method. You make a fresh snapshot of your Redis instance and copy it over to Valkey. Valkey reads the data from the snapshot on startup and restores its contents into memory. The tradeoffs for this method are:
+
+* The downtime to shutdown Redis and wait for Valkey to load the data. 
+* Potential risk of data loss on instances with heavy writes. To prevent it, you must disconnect all active connections before the migration start.
+
+1. Disconnect all active connections to the Redis instance.
+
+2. Make a backup of your Redis instance. 
+
+   a. Connect to Redis and check the number of keys:
+
+      ```
+      $ redis-cli -h 127.0.0.1 -p 6379
+      redis> INFO KEYSPACE
+      # Keyspace
+      db0:keys=6286,expires=0,avg_ttl=0
+      ```
+    
+   b. Check the configuration for the directory (`dir`) where Redis stores its database files and the name of the database file (`dbfilename`):
+
+       ```
```

Comment:
This one is still rendering with the backticks. 

<img width="841" alt="Screenshot 2024-09-10 at 1 34 25 PM" src="https://github.com/user-attachments/assets/81b4441c-7212-4ab6-8b07-ba3e15494448">

If you want to check, the preview requires a little bit of work to setup but you can find the how to here: https://github.com/valkey-io/valkey-io.github.io?tab=readme-ov-file#building-the-documentation-topics


Otherwise LGTM.

Reply:
Fixed intendation, checked the output, seems to be OK